src/main/java/us/icarenow/web/model/entity/User.java

```diff
@@ -0,0 +1,48 @@
+package us.icarenow.web.model.entity;
+
+import javax.persistence.*;
+
+@Entity
+@Table(name="icn_user")
+public class User {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private int id;
+
+    private String email;
+
+    private String password;
```

Comment:
daca am inteles bine de la Andrei si chiar trebuie sa puste denumirile cu denumirile din sql, aici cred ca o sa trebuiasca sa pui icn_password ca "password" e keyword in sql

Reply:
doar la password

Reply:
Bun

Reply:
nu, wait, campurile nu trebuie sa fie icn_, doar tabelele

Reply:
ok, not changing here

Reply:
please wait for Adi's changes. Once his changes are in main branch then update your branchand adapt to his changes